subnet/deployment-generator/script/gen_other.js

```diff
@@ -0,0 +1,145 @@
+
+const config = require('./config_gen')
+Object.freeze(config)
+
+module.exports = { 
+  genSubnetKeys,
+  genDeploymentJson,
+  genDeploymentJsonMac,
+  genCommands,
+  genGenesisInputFile,
+};
+
+
+function genSubnetKeys(){
+  const keys = {}
+  num = config.keys.subnets_addr.length
+  for(i=0; i<config.keys.subnets_addr.length; i++){
+    const key = `key${i+1}`
+    const private_key = config.keys.subnets_pk[i]
+    const address = config.keys.subnets_addr[i]
+    keys[key] = {
+      "PrivateKey": private_key,
+      "0x": address,
+      "short": address.replace(/^0x/i, '')
+    }
+  }
+  keys['Grandmaster'] = {
+    "PrivateKey": config.keys.grandmaster_pk,
+    "0x": config.keys.grandmaster_addr,
+    "short": config.keys.grandmaster_addr.replace(/^0x/i, '')
+  }
+  if (Object.keys(keys).length != config.num_subnet+1){      //sanity check
+    console.log('bad case, key length not equal number of subnets')
+    console.log(Object.keys(keys).length, config.num_subnet+1)
+    console.log(keys)
+    exit()
+  }
+
+  return keys
+}
+
+function genDeploymentJson(keys){
+  num = Object.keys(keys).length-1;
+  validators = []
+  for (let i=1; i<= num; i++){
+    key_name = `key${i}`
+    public_key = keys[key_name]['0x']
+    validators.push(public_key)
+  }
+
+  deployment = {
+    "validators": validators,
+    "gap": 450,
+    "epoch": 900,
+    "xdcparentnet": "https://devnetstats.apothem.network/devnet",
```

Comment:
need to make this as variable right?

Reply:
huh I never noticed this is fixed. Anyways in new CSC galaxy moved this config to different file. Will change next PR.